src/main/java/org/example/model/builders/UserModelBuilder.java

```diff
@@ -0,0 +1,59 @@
+package org.example.model.builders;
+
+import org.example.model.AccountModel;
+import org.example.model.UserModel;
+
+public class UserModelBuilder {
+    private int id;
+    private String username;
+    private String password;
+    private String name;
+    private String surname;
+    private String phoneNumber;
+    private String emailAddress;
+    private String account;
+
+    public UserModelBuilder setId(int id) {
+        this.id = id;
+        return this;
+    }
+
+    public UserModelBuilder setUsername(String username) {
+        this.username = username;
+        return this;
+    }
+
+    public UserModelBuilder setPassword(String password) {
+        this.password = password;
+        return this;
```

Comment:
Kodel seteris grazina visa klase?

Reply:
Panasu bande Builderi pakurti. Tai Buileriui nereikia setXXXXX uztenka kontretinti klases duomeni na ir apgal patterna nustatyti taisykle kad klase turi buileri (per inner static class), vienas is paprastu pvz:
https://dzone.com/articles/design-patterns-the-builder-pattern   ziur. nuo **Pattern**